ci/docker/Dockerfile.build.ubuntu

```diff
@@ -110,6 +108,15 @@ COPY install/requirements /work/
 RUN python3 -m pip install cmake==3.16.6 && \
     python3 -m pip install -r /work/requirements
 
+RUN git clone --recursive -b 3.5.1.1 https://github.com/google/protobuf.git && \
+    cd protobuf && \
+    ./autogen.sh && \
+    ./configure && \
+    make -j$(nproc) install && \
+    cd .. && \
+    rm -rf protobuf && \
+    ldconfig
```

Comment:
Is the version provided by Ubuntu too old?

Reply:
I am not sure, but onnx is failing and the error seems to be related to the protobuf version.

This is the version we are using in our internal containers

Reply:
Do you know that you can reproduce the CI build via `python ci/build.py -R --platform ubuntu_gpu_cu102 /work/runtime_functions.sh build_ubuntu_gpu_tensorrt`?